jvm/src/main/scala/org/json4s/JValue.scala

```diff
@@ -0,0 +1,48 @@
+package org.json4s
```

Comment:
The previous version had this in `org.json4s.ast`.  That's a little more verbose, but would allow the json4s reboot to live alongside the legacy version for migration.

I don't have a strong opinion, but a strategy is needed here.